site/_extensions/gallery_generator.py

```diff
@@ -68,7 +68,7 @@ def generate_repo_dicts(all_items):
             gallery_info_dict = yaml.safe_load(urllib.request.urlopen(gallery_info_url))
             thumbnail = gallery_info_dict["thumbnail"]
             tag_dict = {
-                k: v for k, v in gallery_info_dict["tags"].items() if v[0] is not None
+                k: v for k, v in gallery_info_dict["tags"].items() if v is not None
```

Comment:
Do we want to check for both `v is not None` and `v[0] is not None`?

Reply:
Can't hurt. I just implemented that in the latest push.